src/js/pages/services/ServicesTab.js

```diff
@@ -108,6 +110,15 @@ var ServicesTab = React.createClass({
 
     // Find item in root tree and default to root tree if there is no match
     let item = DCOSStore.serviceTree.findItemById(id) || DCOSStore.serviceTree;
+    let groupId = ServicesPathUtils.getGroupFromAppId(item.id);
```

Comment:
Please use the correct getter `item.getId()` instead of accessing the property directly. The `item.id` property is only avialable due to a scruffy item struct implementation.